addons/project_todo/views/project_task_views.xml

```diff
@@ -7,7 +7,7 @@
         <field name="model">project.task</field>
         <field name="priority">800</field>
         <field name="arch" type="xml">
-            <kanban default_group_by="personal_stage_type_ids"
+            <kanban default_group_by="personal_stage_type_id"
```

Comment:
> Stupid question, if we don't do that change, it will work as well, right? 🧐
> 
> (It is just to be sure if I correctly understood)

It will work but you won't get the possibility to change records from personal stages or to create personal stages directly from the kanban view as it is disable if it is grouped by a M2M field.

Reply:
Ahhh okay, I see, thanks. 🙂